sip-core/src/main/resources/sip-core-default-config.yaml

```diff
@@ -10,6 +10,7 @@ sip.core:
     exchange-formatter:
       showHeaders: true
       showExchangeId: true
+    traceType: LOG
```

Comment:
In docs it is written that default is BOTH, let's make it match, either a change here or in docs.

Reply:
agree, done